udata/tests/dataset/test_dataset_events.py

```diff
@@ -77,7 +77,7 @@ def test_publish_message_resource_modified(self, mock_req):
             dataset.update_resource(resource)
 
         mock_req.assert_called_with(
-            f"{current_app.config['RESOURCES_ANALYSER_URI']}/api/resource/updated/",
+            f"{current_app.config['RESOURCES_ANALYSER_URI']}/api/resources/",
```

Comment:
For information, it's not part of this issue but at some point I'd like to make the _routes_ really REST, meaning on this call it would be a `PUT` on `/api/resources/{resource_id}`, with still the same payload.

Does that make sense to you?

Either we prepare it in this PR, either in another one, as you wish.

Reply:
I think it would make sense to do this in two steps to ease the "synchronization" of the PRs in udata and hydra. So i think i'd rather merge this PR as is, if it looks good to you, and then create two new linked issues in udata and hydra, that'll be prioritized and fixed later.

Reply:
Yes I agree!